src/molecules/hero-image/hero-image.tsx

```diff
@@ -2,41 +2,31 @@ import React from "react";
 import { HeroImageBaseTypes } from "./types";
 import { HeroImageMetadata } from "../../types/story";
 import styled from "styled-components";
-import { media } from "../../utils/media";
 import { withStoryAndConfig } from "../../context";
 import { Image } from "../../atoms";
 import get from "lodash.get";
 
-const StyledWrapper = styled.figcaption`
-  display: flex;
-  align-items: center;
-  text-align: left;
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  padding: 8px 0;
-  max-height: 90px;
-  color: ${(props) => props.theme.color.white};
-  line-height: ${(props) => props.theme.font.lineHeight.level1};
-  background: ${(props) => `${props.theme.color.black}60`};
+const StyledCaption = styled.span`
+  margin-right: 8px;
+  color: ${(props) => props.theme.color.mono6};
+  line-height: ${(props) => props.theme.font.lineHeight.level3};
   font-size: ${(props) => props.theme.font.size.xxs};
-  z-index: ${(props) => props.theme.zIndex.z100};
-
-  ${media.laptop`
-		max-height: 130px;
-		overflow-y: scroll;
-	`}
+  word-wrap: break-word;
```

Comment:
can use `overflow-wrap` instead of `word-wrap`
`word-wrap` is a non standard css property

Reply:
We can use it. But the same property is used in several places. if I change, I will be changing only this and not other places even though its not consistent as it will need another round of testing.

Reply:
okay.. but let's try using it in new changes henceforth

Reply:
Sure